src/Illuminate/Validation/Rules/RequiredIf.php

```diff
@@ -10,18 +11,18 @@ class RequiredIf implements Stringable
     /**
      * The condition that validates the attribute.
      *
-     * @var callable|bool
+     * @var \Closure|bool
      */
     public $condition;
 
     /**
      * Create a new required validation rule based on a condition.
      *
-     * @param  callable|bool  $condition
+     * @param  \Closure|bool  $condition
```

Comment:
Would it make sense to narrow this further down?
```suggestion
     * @param  (\Closure(): bool)|bool  $condition
```